File: backend/internal/science/muscle_groups.go
package science

// MuscleGroups is the fixed 9-value taxonomy shared by exercises' muscle
// maps, goal targets, and the mobile app's heatmap — must match the CHECK
// constraints on the muscle-typed columns in migrations 000010 and 000015.
var MuscleGroups = []string{
	"GLUTES", "QUADS", "HAMSTRINGS", "CHEST", "BACK", "SHOULDERS", "ARMS", "CORE", "CALVES",
}

File: backend/src/science/muscleGroups.ts
import { MuscleGroup } from '@prisma/client';

export const MUSCLE_GROUPS: { value: MuscleGroup; label: string }[] = [
  { value: MuscleGroup.GLUTES, label: 'Glutes' },
  { value: MuscleGroup.QUADS, label: 'Quads' },
  { value: MuscleGroup.HAMSTRINGS, label: 'Hamstrings' },
  { value: MuscleGroup.CHEST, label: 'Chest' },
  { value: MuscleGroup.BACK, label: 'Back' },
  { value: MuscleGroup.SHOULDERS, label: 'Shoulders' },
  { value: MuscleGroup.ARMS, label: 'Arms' },
  { value: MuscleGroup.CORE, label: 'Core' },
  { value: MuscleGroup.CALVES, label: 'Calves' },
];
